pcsx2/SIO/Pad/PadDualshock2.cpp

```diff
@@ -71,25 +71,25 @@ static const SettingInfo s_settings[] = {
 	{SettingInfo::Type::Float, "Deadzone", TRANSLATE_NOOP("Pad", "Analog Deadzone"),
 		TRANSLATE_NOOP(
 			"Pad", "Sets the analog stick deadzone, i.e. the fraction of the stick movement which will be ignored."),
-		"0.00", "0.00", "1.00", "0.01", "%.0f%%", nullptr, nullptr, 100.0f},
+		"0.00", "0.00", "1.00", "0.01", TRANSLATE_NOOP("Pad", "%.0f%%"), nullptr, nullptr, 100.0f},
```

Comment:
These likely won't work, because Qt doesn't know to translate the format parameter.

Reply:
Is there any way to solve this?

Reply:
Done in https://github.com/PCSX2/pcsx2/commit/e18205e6c72f4e1fb096d619fdb58274b137837e

Reply:
Thank you!